Contributing.md

```diff
@@ -2,27 +2,34 @@
 
 WebAssembly is initially designed and implemented by browser vendors who are
 interested in meeting a variety of [use cases](UseCases.md). As design and
-implementation progresses they'll need input and contributions from developers
+implementation progresses they need input and contributions from developers
 interested in using WebAssembly.
 
 Interested in participating? We suggest you start by:
 
 1. Acquainting yourself with the
    [Code of Ethics and Professional Conduct](CodeOfConduct.md).
-2. Reading the [WebAssembly design][].
+2. Reading the WebAssembly [design documents][] and the [developer guide][].
 3. Joining the IRC channel `irc://irc.w3.org:6667/#webassembly`.
+4. Perusing the [roadmap][].
 
-With that background understood and communication set up, feel free to
-[file issues][] in the WebAssembly design repository. Please join the
-[W3C Community Group][] before sending pull requests: it provides the legal
-framework that protects the work in this repository. Make sure you're affiliated
-with your company or organization in the Community Group, if any.
+With that background understood and communication set up, feel free to:
 
-As WebAssembly moves forward we expect to form an official standards body, which
+ - [file issues][] in the WebAssembly design repository
+ - [participate][] in browser implementation and tooling development
+
+Please join the [W3C Community Group][] before sending pull requests: it provides
+the legal framework that protects the work in this repository. Make sure you're 
+affiliated with your company or organization in the Community Group, if any.
+
+As WebAssembly moves forward we will form an official standards body, which
 will have its own contribution process to the specification.
 
 Happy assembly!
 
-  [WebAssembly design]: https://github.com/WebAssembly/design
+  [design documents]: https://github.com/WebAssembly/design
+  [developer guide]: http://webassembly.org/getting-started/developers-guide/
+  [roadmap]: http://webassembly.org/roadmap/
```

Comment:
The roadmap is part of the Community Group: https://github.com/WebAssembly/website/blob/master/roadmap.md